personal/privacyidea/tokens/class_MFATokenHelper.inc

```diff
@@ -133,6 +133,8 @@ class MFATokenHelper
         /* START COMPARISONS */
         // Skip comparison if token limit is 0. That means there is no maximum limit for tokens.
         if ($tokenLimitAll != 0) {
+            $this->utils->tokenCounts["all"] = $this->utils->tokenCounts["all"] ?? 0;
```

Comment:
Does this also cover the case where the array $this->utils->tokenCounts does not have the element tokenCounts["all"] set?

Reply:
This is exactly what this line does. If its not set, then set it to 0.